--- a/target/linux/econet/base-files/sbin/en75_chboot
+++ b/target/linux/econet/base-files/sbin/en75_chboot
@@ -145,6 +145,15 @@
         code_openwrt=30
         code_factory=31
         ;;
+    zyxel,vmg8825-t50)
+        # Ipotesi: identico a EX3301-T0 finche non confermato con boot log.
+        part=$(part_named '"reservearea"')
+        offset_blocks=5
+        block_size=$((1024 * 128))
+        code_offset=4095
+        code_openwrt=30
+        code_factory=31
+        ;;
     zyxel,pmg5617ga)
         # 00060fff
         part=$(part_named '"reservearea"')
